src/main/java/cloud/fogbow/ras/core/plugins/interoperability/cloudstack/securityrule/v4_9/CloudStackSecurityRulePlugin.java

```diff
@@ -97,6 +88,24 @@ public void deleteSecurityRule(String securityRuleId, @NotNull CloudStackUser cl
         doDeleteInstance(request, cloudStackUser);
     }
 
+    @NotNull
+    @VisibleForTesting
+    List<SecurityRuleInstance> doGetSecurityRules(@NotNull Order majorOrder,
+                                                  @NotNull CloudStackUser cloudStackUser)
+            throws FogbowException {
+
+        switch (majorOrder.getType()) {
+            case PUBLIC_IP:
+                String publicIpId = CloudStackPublicIpPlugin.getPublicIpId(majorOrder.getId());
+                return getFirewallRules(publicIpId, cloudStackUser);
+            case NETWORK:
+                return new ArrayList<>();
```

Comment:
there isn't security rules for network in cloudstack?

Reply:
Yeap, there isn't it.